pandas/core/generic.py

```diff
@@ -9862,10 +9862,10 @@ def _add_numeric_operations(cls):
         axis_descr, name, name2 = _doc_parms(cls)
 
         cls.any = _make_logical_function(
-            cls, 'any', name, name2, axis_descr,
+            cls, 'any', name, name2, False, axis_descr,
```

Comment:
I can't guess what's the `False` without checking the function signature, we should use keyword arguments.

Reply:
Would you prefer me to move the argument to the end of the list, or add keywords to all of the arguments?

Reply:
you can add the argument names to all params, or to all the ones after the `True`/`False`, whatever you think it's clearer.